CONTRIBUTING.md

```diff
@@ -0,0 +1,17 @@
+# Contributing
+
+Optimally, when contributing to this repository, please first discuss the change you wish to make via an [issue](https://github.com/GIScience/ohsome-examples/issues/new), [email](mailto:ohsome@heigit.org), or any other method with the owners of this repository before making a change.
```

Comment:
Can the `owners of this repository` be linked to the GitHub team @GIScience/ohsome in any useful way? Or do you want to link to [ohsome.org](https://ohsome.org) or something similar?